static/electrical_foot_print.csv

```diff
@@ -1,33 +1,33 @@
-Germany,DEU,2020,0.301,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Greece,GRC,2020,0.522,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Hungary,HUN,2020,0.218,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Ireland,IRL,2020,0.293,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Italy,ITA,2020,0.212,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Latvia,LVA,2020,0.092,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Lithuania,LTU,2020,0.146,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Luxembourg,LUX,2020,0.069,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Malta,MLT,2020,0.356,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Netherlands,NLD,2020,0.318,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Poland,POL,2020,0.724,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Portugal,PRT,2020,0.201,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Romania,ROU,2020,0.208,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Slovakia,SVK,2020,0.09,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Slovenia,SVN,2020,0.219,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Spain,ESP,2020,0.19,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-Sweden,SWE,2020,0.013,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
-United States,USA,2022,0.373,0,0,https://www.epa.gov/egrid/download-data
-UnitedKingdom,GBR,2020,0.209,0,0,https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html;https://ember-climate.org/data/
\ No newline at end of file
+Austria,AUT,2022,0.158,0,0,'Ember Climate, CO2.js'
```

Comment:
for my own understanding, intensity has doubled in 2 years?

Reply:
Yeah I noticed some wild changes in the data as well. In this case, when checking in Ember for the year 2020 they report a grid intensity for Austria of 142.57 gCO2e per kWh. I also checked the BP data that's linked in the current file & can't find anything that corresponds to the `0.083` value. 

I know that Ember updates their data as new data/information comes in, so perhaps there's been an update to the data since the original CSV file was published? That's purely speculation though.

Reply:
Thanks